src/libraries/StateLibrary.sol

```diff
@@ -3,9 +3,10 @@ pragma solidity ^0.8.21;
 
 import {PoolId} from "../types/PoolId.sol";
 import {IPoolManager} from "../interfaces/IPoolManager.sol";
+import {Currency} from "../types/Currency.sol";
 import {Position} from "./Position.sol";
 
-library PoolStateLibrary {
+library StateLibrary {
```

Comment:
maybe we actually rename this back to PoolStateLibrary? I think there may end up being more libraries we need for various view function in PoolManager (ie for some view functions in ProtocolFee.sol) and we might want them in their own lib?

Reply:
will just leave for now